src/components/atoms/UiTextarea/UiTextarea.vue

```diff
@@ -181,8 +192,10 @@ defineExpose(
     @include mixins.use-logical($element, padding, var(--space-12) var(--space-16));
     @include mixins.use-logical($element, border, 0);
 
+    margin: var(--space-2);
     overflow: hidden;
-    max-width: 100%;
+    overflow-y: scroll;
```

Comment:
```suggestion
    overflow-y: auto;
```
why we need scroll "for empty textarea"? 
![Zrzut ekranu 2024-05-15 o 08 08 39](https://github.com/infermedica/component-library/assets/12138170/b04597fe-abe0-4457-ab67-f79d59ea6836)